Bio/AlignIO/FastaIO.py

```diff
@@ -132,7 +134,7 @@ def build_hsp():
             print(match_tags)
             print("%s %i" % (m, len(m)))
             print(handle.name)
-            raise err
+            raise ValueError("Darn... amino acids vs nucleotide coordinates?")
```

Comment:
Since you are working on this bit of code, could you move the information in the above debugging print statements into the exception message?

Reply:
Yes, definitely. I was going to ask about this.